app/webpack/javascripts/modules/Modules.AllocationDataTable.js

```diff
@@ -56,9 +56,13 @@ moj.Modules.AllocationDataTable = {
         $('td', row).eq(0).wrapInner("<div class='error-message-container'></div>");
         $('td .error-message-container', row).eq(0).append("<div class='error-message'>" + data.injection_errors + "</div>");
       } else if (data.filter.cav_warning) {
-        $(row).addClass('cav_warning');
+        $(row).addClass('injection_warning');
```

Comment:
@kmahern Do you wish to apply the same style to cav and scheme_12_fees warnings?

Reply:
@naseberry that was the idea, unless there is a reason for having different styles?